tests/integration_test/admin_controller.py

```diff
@@ -450,6 +453,71 @@ def execute_actions(self, site_launcher, actions):
                         client_ids = list(site_launcher.client_properties.keys())
                     for cid in client_ids:
                         site_launcher.start_client(cid)
+
+            elif command == "test":
+                if args[0] == "admin_commands":
+                    print(("\n" + "*" * 120) * 20)
```

Comment:
another suggestion can you create a function something like

`admin_api_test(admin_api)`

And move these logic inside the function, and we just pass self.admin_api as argument

Reply:
done